deploy/crds/objectbucket_v1alpha1_objectbucketclaim_crd.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectbucketclaims.objectbucket.io
+spec:
+  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  group: objectbucket.io
+  names:
+    kind: ObjectBucketClaim
+    listKind: ObjectBucketClaimList
+    plural: objectbucketclaims
+    singular: objectbucketclaim
+    shortNames:
+      - obc
+      - obcs
+  scope: Namespaced
+  subresources:
+    status: {}
+  additionalPrinterColumns:
+  - JSONPath: .spec.storageClassName
+    description: StorageClass
+    name: Storage-Class
+    type: string
+  - JSONPath: .spec.claimRef.namespace
+    description: ClaimNamespace
+    name: Claim-Namespace
+    type: string
+  - JSONPath: .spec.claimRef.name
+    description: ClaimName
+    name: Claim-Name
+    type: string
+  - JSONPath: .spec.reclaimPolicy
+    description: ReclaimPolicy
+    name: Reclaim-Policy
+    type: string
+  - JSONPath: .status.phase
+    description: Phase
+    name: Phase
+    type: string
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
```

Comment:
I know there's a printcolumns pr related to this field, but on the surface it seems that Age is not the same as a create timestamp w/o doing some math.

Reply:
It works correctly.